src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -11,6 +11,12 @@
       CS8002: Referenced assembly does not have a strong name
     -->
     <NoWarn>$(NoWarn);CS1591;CS8002</NoWarn>
+
+    <!-- This Application is a framework-dependent application that targets .NET 8.0. This means that the
+    application is not able to run if the place where it is deployed does not have the .NET 8.0 runtime installed.
+    Given we want to be able to support people running on environments where they only have the 9.0 SDK/runtime installed,
+    we allow roll-forward to the next major in order to support these customers.-->
+    <RollForward>Major</RollForward>
```

Comment:
How are we going to test this?

Reply:
I wrote in the description, the plan is that once we add 9.0 templates (@radical  is working on it) we'll be adding tests specifically for environments with only 9.0 SDK installed (to validate that 9.0 templates can be created and built), so at that time we'll be able to add a test for this by using the 9.0 SDK to create a template, run it, and then validate that the dashboard is loading correctly.

Reply:
Do we also want to run the Dashboard tests on net9? "validate that the dashboard is loading correctly" is a good first step, but it won't cover all the use cases of the dashboard.

Reply:
We can certainly consider it, but not sure if we would get a lot from it so perhaps we can consider doing it in a format similar to runtime's outerloop runs (run once a day maybe?). Point being those tests would mostly be trying to catch breaking changes between 8.0 and 9.0, which while they are very possible, I'm guessing it is not something that may add a lot of value on testing on each CI run.

Reply:
My above suggestion for "is it loading correctly" was simply to validate that the property is doing it's job and the 9.0 runtime is effectively being able to run the dashboard to begin with.